tasks/eth/ink-001-permissionless-proofs/.env

```diff
@@ -0,0 +1,14 @@
+ETH_RPC_URL="https://ethereum.publicnode.com"
+COUNCIL_SAFE=0xc2819DC788505Aac350142A7A707BF9D03E3Bd03
+FOUNDATION_SAFE=0x847B5c174615B1B7fDF770882256e2D3E95b9D92
+SAFE_NONCE=6
```

Comment:
Let's make sure to update the nonce env vars per our notion tracker doc, I believe this should be 7, and there are two other `SAFE_NONCE_*` env vars we need